submitty.sh

```diff
@@ -2,3 +2,5 @@
 export REPO_DIR=/usr/local/submitty/GIT_CHECKOUT/AnalysisToolsTS
 export INSTALLATION_DIR=/usr/local/submitty/SubmittyAnalysisToolsTS
 export INCLUDE_DIR="${INSTALLATION_DIR}"/include
+export NLOHMANN_DIR=/usr/local/submitty/GIT_CHECKOUT/vendor/nlohmann/json
```

Comment:
Perhaps you could make another variable for the Submitty installation directory `/usr/local/submitty`